project.clj

```diff
@@ -6,14 +9,32 @@
 
   :dependencies [[org.clojure/clojure "1.6.0"]
                  [org.clojure/tools.reader "0.8.9"]
+
                  [ring/ring-core "1.3.2"]
+                 [clj-time "0.7.0"]
                  [commons-io "2.4"]
+                 [commons-codec "1.9"]
+                 [slingshot "0.12.2"]
+                 [org.slf4j/slf4j-api "1.7.7"]
+
                  [bidi "1.18.9" :exclusions [org.clojure/clojurescript]]
                  [compojure "1.3.3"]
                  [prismatic/schema "0.4.0"]
-                 [puppetlabs/kitchensink "1.1.0"]]
+                 [puppetlabs/kitchensink ~ks-version]
+                 [puppetlabs/trapperkeeper-metrics "0.1.1-SNAPSHOT"]]
```

Comment:
In general, I'd like to avoid introducing this dependency on Trapperkeeper based on the notion that libraries should have as few dependencies as possible, making them easier to consume.  I figure you've already considered that - and I recall getting into a discussion about this issue on other projects in the past, so we don't need to re-hash - just wanted to throw it out there.